mix.exs

```diff
@@ -104,7 +105,8 @@ defmodule Skate.MixProject do
       "assets.setup": ["cmd npm --prefix=assets install"],
       "assets.reset": ["cmd npm --prefix=assets ci"],
       test: ["ecto.create --quiet", "ecto.migrate_all", "test"],
-      fmt: ["format", "cmd npm --prefix=assets run format"]
+      fmt: ["format", "cmd npm --prefix=assets run format"],
+      sobelow: "sobelow --skip --exit=low --ignore Config.HTTPS,Config.CSWH"
```

Comment:
question: what is the rationale for ignoring CSWH / can we address the issues?

Reply:
The reason I picked this approach for ignoring these errors is because `# sobelow_skip` only works on functions, and the `HTTPS` and `CSWH` issues were flagged in `router.ex`, which doesn't have much in the way of functions, and I couldn't figure out how to actually _resolve_ them.

There is another option, which is to use the `--mark_skip_all` flag to generate a `.sobelow_skips` file. The problem with that is that `.sobelow_skips` is kind of incomprehensible.

Reply:
And for `CSWH` in particular, I'm reasonably certain that it's already not an issue, because we do have `check_origin` [set for websockets](https://github.com/mbta/skate/blob/7c0347fab359c43b849a63033e7cb8cb37499651/lib/skate_web/endpoint.ex#L5-L7), and that environment config is [always set for prod environments](https://github.com/mbta/skate/blob/7c0347fab359c43b849a63033e7cb8cb37499651/config/prod.exs#L36-L39), but I don't know how to convince Sobelow that that's true. Hence the ignore here.